### DISCUSSION TITLE
storage/hwameistor/intro/resources/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1671] [Announcements]
# storage/hwameistor/intro/resources/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/intro/resources/

**Comment by liyuerich** [comment]
能给出怎么查看这些类的命令吗？

**Reply by windsonsea** [reply]
Added an example in PR #5350
